test/single-issuer.c

```diff
@@ -56,13 +55,13 @@ static int try_submit(struct io_uring *ring)
 		return ret;
 
 	if (ret != 1)
-		error(1, ret, "submit %i", ret);
+	t_error(1, ret, "submit %i", ret);
```

Comment:
Also missing a tab here, and basically everywhere else too.

Reply:
Yeah I just recognized it here in my browser D: